src/resultsdisassemblypage.cpp

```diff
@@ -244,3 +278,18 @@ void ResultsDisassemblyPage::jumpToAsmCallee(QModelIndex index)
     }
     showDisassembly();
 }
+
+void ResultsDisassemblyPage::setNoShowRawInsn(bool noShowRawInsn)
+{
+    m_noShowRawInsn = noShowRawInsn;
+}
+
+void ResultsDisassemblyPage::setNoShowAddress(bool noShowAddress)
+{
+    m_noShowAddress = noShowAddress;
+}
+
+void ResultsDisassemblyPage::setIntelSyntaxDisassembly(bool intelSyntax)
```

Comment:
setDisassemblySyntax(DisassemblySyntax syntax)

make the DisassemblySyntax an enum with Intel and ATT enumerators